### PR TITLE
Minor code fixes

### PR DESCRIPTION
**Pull request by tariq1890**
Fixing some linting errors reported by gometalinter.

### CODE REVIEW

**Comment by tariq1890** [comment]
@shlomi-noach can this be merged?

File: go/cmd/ccql/main.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func main() {
 
 	if *askPassword {
 		fmt.Print("Mysql password: ")
-		passwd, err := terminal.ReadPassword(int(syscall.Stdin))
+		passwd, err := terminal.ReadPassword(syscall.Stdin)
```
Comment (shlomi-noach) [inline go/cmd/ccql/main.go]: 👍

File: go/logic/ccql.go [hunk 2]
```diff
@@ -38,8 +42,9 @@ func queryHost(host string, user string, password string, schema string, queries
 }
 
 // QueryHosts will issue concurrent queries on given list of hosts
-func QueryHosts(hosts []string, user string, password string,
-	defaultSchema string, schemas []string, queries []string,
+func QueryHosts(
+	hosts, schemas, queries []string,
+	user, password, defaultSchema string,
```
Comment (shlomi-noach) [inline go/logic/ccql.go]: I suspect the reason for reordering arguments was by their data types. I disagree on the reordering. Data types do not have semantics, I'd like ordering to return to what it used to be, consistent with `queryHost()`

File: go/text/hosts.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ func ParseHosts(hostsList string, hostsFile string) (hosts []string, err error)
 		}
 		hostsList = string(bytes)
 	}
-	parsedHosts := regexp.MustCompile("[,\\s]").Split(hostsList, -1)
+	parsedHosts := regexp.MustCompile(`[,\s]`).Split(hostsList, -1)
```
Comment (shlomi-noach) [inline go/text/hosts.go]: :+1:

File: go/logic/ccql.go [hunk 4]
```diff
@@ -11,7 +11,11 @@ import (
 
 // queryHost connects to a given host, issues the given set of queries, and outputs the results
 // line per row in tab delimited format
-func queryHost(host string, user string, password string, schema string, queries []string, timeout float64, printSchema bool) error {
+func queryHost(
+	host, user, password, schema string,
+	queries []string,
+	timeout float64, printSchema bool,
```
Comment (shlomi-noach) [inline go/logic/ccql.go]: Thank you for this work. Line breaks are fine, but also a matter of opinion.

**Comment by tariq1890** [comment]
@shlomi-noach Done. Thanks for the review :)

**Comment by tariq1890** [comment]
@shlomi-noach Can this be merged?

**Pull request merged**